### PR TITLE
wr-81: Fix resources fetching on landing page

### PR DESCRIPTION
**Pull request by kovAnya**

### CODE REVIEW

File: frontend/src/pages/landing/styles.module.scss [hunk 1]
```diff
@@ -28,3 +27,12 @@
   text-align: center;
   text-transform: uppercase;
 }
+
+.lettersImg {
+  z-index: -5;
+}
+
+.typewriterImg {
+  z-index: -10;
```
Comment (andanf-e) [inline frontend/src/pages/landing/styles.module.scss]: Not sure why do we need negative indexing

File: frontend/src/pages/landing/landing.tsx [hunk 2]
```diff
@@ -1,37 +1,72 @@
...
       <Link className={styles.signLink} to={AppRoute.SIGN_IN}>
         Unbounded space for freedom of your feather
       </Link>
+
+      <img
+        ref={lettersElementReference}
+        className={`${styles.parallaxImg} ${styles.lettersImg}`}
+        src={backgroundLetters}
+        alt="letters"
+      ></img>
```
Comment (artemmatiushenko1) [inline frontend/src/pages/landing/landing.tsx]: We can use a self-closing tag instead, like ``

File: frontend/src/pages/landing/styles.module.scss (hunk 1 above)
Comment (kovAnya) [reply frontend/src/pages/landing/styles.module.scss]: Really, it wasn't a good decision. Fixed it

File: frontend/src/pages/landing/landing.tsx (hunk 2 above)
Comment (kovAnya) [reply frontend/src/pages/landing/landing.tsx]: Thanks, fixed it

File: frontend/src/pages/landing/styles.module.scss [hunk 3]
```diff
@@ -5,20 +5,20 @@
...
+
+.parallaxImg {
+  position: fixed;
+  top: 50%;
+  left: 50%;
   transform: translate(-50%, -50%);
 }
 
 .signLink {
+  z-index: 5;
```
Comment (andanf-e) [inline frontend/src/pages/landing/styles.module.scss]: Let's use defined z-index from variables, or add this one to variables if necessary

**Review by andanf-e — APPROVED** [review]

**Review by mhrechkin — APPROVED** [review]

**Review by NazarKovtun1 — APPROVED** [review]

**Pull request merged**